mod/embed/start.php

```diff
@@ -82,7 +82,10 @@ function embed_longtext_menu($hook, $type, $items, $vars) {
  */
 function embed_select_tab($hook, $type, $items, $vars) {
 	$tab_name = elgg_extract('tab', $vars);
```

Comment:
better would be `$tab_name = elgg_extract('tab', $vars, 'file');`

Reply:
Thanks for your guidance.